### PR TITLE
Resolve docker build failures by upgrading Java and installing pydot via apt-get

### PR DESCRIPTION
**Pull request by jmthibault79**
Not clear why this became a problem today.  The real solution is #148 but this bandaid solves today's problems.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: jupyter-docker/Dockerfile [hunk 1]
```diff
@@ -147,6 +149,8 @@ RUN apt-get install -yq --no-install-recommends python3 \
  && pip3 install seaborn \
  && pip3 install python-lzo \
  && pip3 install google-cloud \
+ # pydot is a dependency of firecloud; peg the version because pydot >=1.2.2 requires a version of pyparsing which conflicts with the distribution
+ && pip install 'pydot<1.2.2' \
```
Comment (rtitle) [inline jupyter-docker/Dockerfile]: Should this one be pip3?

**Comment by jmthibault79** [comment]
Working on a solution for python 3 but it's not ready yet.  TLDR: install the system package `python3-pydot`

File: jupyter-docker/Dockerfile (hunk 1 above)
Comment (jmthibault79) [reply jupyter-docker/Dockerfile]: yes but I've ditched in this in favor of a better fix

**Comment by jmthibault79** [comment]
Blech.  Great that we can see it on this PR, though.

**Comment by jmthibault79** [comment]
Here's one more attempt.  But now I should get back to what I'm supposed to be working on...  :)

File: jupyter-docker/Dockerfile [hunk 2]
```diff
@@ -107,6 +107,13 @@ RUN apt-get install -yq --no-install-recommends \
     python-seaborn \
     python-tk \
     python-numpy \
+    python-pydot \
+    python-scipy \
+    python-statsmodels \
```
Comment (jmthibault79) [inline jupyter-docker/Dockerfile]: This one isn't available in Python 3

**Comment by rtitle** [comment]
Thanks. I'll look into this too

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
[leonardo-build](https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-build/1968/) passed, fingers crossed for [leonardo-fiab-test-runner](https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/).

**Comment by jmthibault79** [comment]
nope

**Comment by rtitle** [comment]
Closing for now in favor of https://github.com/DataBiosphere/leonardo/pull/325/ (I hope)

**Pull request closed without merging**